### PR TITLE
`browsingContext.create`: become spec compliant

### PR DESCRIPTION
**Pull request by thiagowfx**
Spec: https://w3c.github.io/webdriver-bidi/#command-browsingContext-create

### CODE REVIEW

**Comment by thiagowfx** [comment]
The WPT tests are incorrect and violate the spec. I'll send a PR upstream.

**Comment by thiagowfx** [comment]
Sent https://github.com/web-platform-tests/wpt/pull/40034

File: src/bidiMapper/domains/context/browsingContextProcessor.ts [hunk 1]
```diff
@@ -230,10 +231,23 @@ export class BrowsingContextProcessor {
...
-      newWindow: params.type === 'window',
-    });
+    let result: Protocol.Target.CreateTargetResponse;
+
+    switch (params.type) {
+      case 'tab':
+        result = await browserCdpClient.sendCommand('Target.createTarget', {
+          url: 'about:blank',
+          newWindow: false,
+          browserContextId: referenceContext?.id ?? undefined,
```
Comment (sadym-chromium) [inline src/bidiMapper/domains/context/browsingContextProcessor.ts:241]: CDP `browserContextId` is not the same as BiDi's `browsingContext`.

**Review by sadym-chromium — COMMENTED** [review]
LGTM, but let's merge the WPT PR first

File: src/bidiMapper/domains/context/browsingContextProcessor.ts (hunk 1 above)
Comment (thiagowfx) [reply src/bidiMapper/domains/context/browsingContextProcessor.ts:241]: Should I revert this snippet? The spec says we should differentiate `tab` and `window` which we don't currently seem to do, but we should try to address it at another time.

Comment (thiagowfx) [reply src/bidiMapper/domains/context/browsingContextProcessor.ts:241]:
```
// TODO: Pass correct browserContextId.
// browserContextId: referenceContext?.id ?? undefined,
```

Would this comment be appropriate?

Comment (sadym-chromium) [reply src/bidiMapper/domains/context/browsingContextProcessor.ts]: In order to implement it properly, we need to:
1. Add `cdpBrowserContextId` to `CdpTarget`.
2. Use it: `browserContextId: referenceContext?.target.cdpBrowserContextId ?? undefined`

Comment (thiagowfx) [reply src/bidiMapper/domains/context/browsingContextProcessor.ts]: Given your previous comment:

> Chrome doesn't support opening tab next to another one, so the parameter should be ignored. No need it TODO.

Omitting this for now.

**Review by sadym-chromium — APPROVED** [review]

**Pull request merged**